fiorella/src/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red: #e83671;
+$text-color-black: #000000;
+
+$playfair-display: "Playfair Display";
```

Comment:
Remember to add a default font here, so that the browser renders that one if it cannot find the main font family, I'll suggest `sans-serif`.